packages/core/core/src/Parcel.js

```diff
@@ -288,9 +273,16 @@ export default class Parcel {
       this.#reporterRunner.report({
         type: 'buildStart',
       });
-      let {assetGraph, changedAssets} = await this.#assetGraphBuilder.build(
-        signal,
-      );
+      let request = createAssetGraphRequest({
+        name: 'Main',
+        entries: nullthrows(this.#resolvedOptions).entries,
+        optionsRef: this.#optionsRef,
+      }); // ? should we create this on every build?
```

Comment:
I guess it's only really necessary if the entries or options change right? But not sure if it really causes any harm to do it every time as long as it doesn't throw away any other nodes...